pulseapi/profiles/models/profiles.py

```diff
@@ -182,6 +182,7 @@ def thumbnail_image_tag(self):
         default=False
     )
 
+    # TODO: Deprecate
```

Comment:
Just to make sure, this would be a followup with a migration function to build the new tuple objects?

Reply:
I don't think it'll be a migration. I guess it's an open question - we can get rid of this at the model level OR, we can just keep it and use versioning to deprecate properties.